lib/Ohmbrewer_Temperature_Sensor.cpp

```diff
@@ -212,3 +311,11 @@ void Ohmbrewer::TemperatureSensor::whichPins(std::list<int>* pins) {
 
 }
 
+/*
+ * publish to the log
+ */
+void log(char* msg)
```

Comment:
If you're going to use this, you need to add the namespace on the function and include the function in the header file. Otherwise you're just inserting a global function, and (despite how this OneWire library works), we're trying to stay fairly object-oriented.